pkg/collector/corechecks/net/wlan/wlan_darwin.m

```diff
@@ -0,0 +1,73 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+#import <CoreWLAN/CoreWLAN.h>
+#import "wlan_darwin.h"
+
+@interface LocationManager : NSObject <CLLocationManagerDelegate>
+
+@property (nonatomic, strong) CLLocationManager *locationManager;
+@property (nonatomic, assign) BOOL isAuthorized;
+
+- (instancetype)init;
+- (BOOL)checkLocationPermissions;
+
+@end
+
+@implementation LocationManager
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.locationManager = [[CLLocationManager alloc] init];
+        self.locationManager.delegate = self;
```

Comment:
Since we are no longer implementing the delegate methods, I think this can be removed (as well as the protocol conformance to `CLLocationManagerDelegate`)

Reply:
Done!